include/xtensor/xutils.hpp

```diff
@@ -612,7 +612,7 @@ namespace xt
      * xtrivial_default_construct implemenation *
      ********************************************/
 
-#if !defined(__GNUG__) || defined(_LIBCPP_VERSION) || defined(_GLIBCXX_USE_CXX11_ABI)
+#if !defined(__GNUG__) || defined(_LIBCPP_VERSION) || (defined(_GLIBCXX_USE_CXX11_ABI) && _GLIBCXX_USE_CXX11_ABI)
```

Comment:
Perhaps this is a partial solution .

Reply:
Why partial?

Reply:
The flag can be set by the user.
You need to check the compiler version .

Reply:
I don't think you need to check the compiler version, `_GLIBCXX_USE_CXX11_ABI` should not be set if GCC < 5.1. Otherwise it is an issue from the user.